layouts/resources/list.html

```diff
@@ -0,0 +1,192 @@
+{{ define "hero" }}
+    <header class="header-hero-dark py-0 pt-16 px-4">
+        <div class="container mx-auto text-white md:flex justify-around items-center">
+            <div class="md:w-1/2">
+                <h1 class="text-white text-center">Resources</h1>
+            </div>
+            <div class="md:w-1/2 md:ml-8">
+                <div class="text-center">
+                    <img class="inline-block w-56" src="/icons/icon-audience-devops.svg">
+                </div>
+            </div>
+        </div>
+    </header>
+
+    <img class="w-full h-20" src="/images/waves.svg">
+{{ end }}
+
+{{ define "main" }}
+    <!-- Set the page context in a variable in a loop. -->
+    {{ $pageContext := . }}
+
+    <!-- Grab the webinar and event page data. -->
+    {{ $webinars := (where $.Site.Pages "Type" "webinars") }}
+    {{ $events := (where $.Site.Pages "Type" "events") }}
+
+    <!-- Create dicts so we can sort the resources by start/publish date. -->
+    {{ $resourcesWithDate := slice }}
+
+    <!-- Create the resource data dicts for webinars. -->
+    {{ range $webinars }}
+        <!-- If the webinar is not marked as unlisted we will add it to the list. -->
+        {{ if eq .Params.unlisted false }}
+            {{ $resourcesWithDate = $resourcesWithDate | append (dict "date" .Params.main.sortable_date "data" .) }}
+        {{ end }}
+    {{ end }}
+
+    <!-- Create the resource data dicts for events. -->
+    {{ range $events }}
+        <!-- If the webinar is not marked as unlisted we will add it to the list. -->
+        {{ if ne .Params.unlisted true }}
+            {{ $resourcesWithDate = $resourcesWithDate | append (dict "date" .Params.main.start_date "data" .) }}
+        {{ end }}
+    {{ end }}
+
+    <div class="pulumi-event-list-container mt-5">
+        <div class="container mx-auto pt-10">
+            <!-- These hidden elements help select the children element so we use CSS -->
+            <!-- to the filter, well filter. -->
```

Comment:
Like the comment, but seemed to be missing a word or two -- hopefully this gets at your meaning, but feel free to adjust/discard as you like!

```suggestion
            <!-- These hidden elements help select the children elements so we can use CSS -->
            <!-- to have the filter, well filter. -->
```

Reply:
👍